crypto/cms/cms_sd.c

```diff
@@ -21,6 +21,7 @@
 #include "crypto/ess.h"
 #include "crypto/x509.h" /* for ossl_x509_add_cert_new() */
 #include "cms_local.h"
+#include "assert.h"
```

Comment:
Please don't use assert().

Reply:
Removed in https://github.com/openssl/openssl/pull/17733/commits/6a6172046810203bff7daecc2d6d0c1df2b8ce93